docs/RFCS/20210415_db_console_cluster_ui.md

```diff
@@ -0,0 +1,74 @@
+
+## Deprecate the package in UI repo
+
+As a final step, the code in `cockroachdb/ui` should be deprecated, the `package.json` set to private, and the Github Action for publishing removed. We should keep the code around for reference or back up until we are confident it is no longer needed.
+
+# Next Step and recommendations for team
+
+## Cluster UI
+
+The Cluster UI team has the most familiarity with the current architecture. To assist in the transition of ownership of these components (to the SQL Observability team) we can commit to doing the bulk of the work mentioned in the above section. Cluster UI would take on moving the code and git commit history into the CRDB repo, updating the build systems, tools, and dependencies required, and configuring npm publishing to allow the SQL <abbr title="observability">o11y</abbr> team to manually publish the package. We could also commit to updating the depedency versions in CockroachCloud console and confirming no unexpected breakage.
+
+## SQL Observability
+
+As the new stewards of the observability components, it is our recommendation to the SQL Observabilty team to consider relocating code specifically engineered for state management and routing in CockroachCloud console to that repo. This should simplify the code in that is pubilshed, the dependency tree, and the code that handles external reducers and routes in that application.
+
+## CockroachCloud Platform
+
+One issue that has been repeatedly encountered by the Cluster UI team in supporting these shared components is resolving conflicts with global styles in CockroachCloud Console. It is our recomendation that the web component paradigmn is best served by removing global css entirely from that application. CC-Console has the ability to use modular css at the component level (CSS Modules). Where styles need to be kept consistent across components, we suggest the use of design tokens or at the very least Sass mixins. 
```

Comment:
which repo those design token will exist?

Reply:
The common design tokens we have currently exist [here](https://github.com/cockroachdb/ui/tree/master/packages/design-tokens).